### PR TITLE
start with mini-stats

### PR DESCRIPTION
**Pull request by slimbuck**
Start documenting the mini stats.

Explains the information shown on screen. Missing is the API and instructions on how to enable the mini stats in game.

### CODE REVIEW

File: content/en/user-manual/optimization/mini-stats.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Mini stats is a graphical display of the application's frame time, CPU load and (where supported) GPU load. Clicking on the mini stats will toggle between the three supported sizes:
+
+<img alt="Mini Stats" width="411" src="/images/user-manual/optimization/mini-stats/mini-stats.gif" />
+
+The information displayed is as follows:
+
+* **GPU** shows the number of milliseconds spent by the GPU rendering the frame
+* **CPU** shows approximate number of milliseconds spent by the CPU
+* **Frame** shows the frame time as passed in by the browser
```
Comment (willeastcott) [inline content/en/user-manual/optimization/mini-stats.md:15]: I'm uncertain what this line means.

Comment (slimbuck) [reply content/en/user-manual/optimization/mini-stats.md]: Is this any better?
"shows the browser's frame time as passed in to the requestAnimationFrame() callback"

Comment (slimbuck) [reply content/en/user-manual/optimization/mini-stats.md]: Or "shows the frame time as calculated by the browser and passed to the requestAnimationFrame() callback"

Comment (willeastcott) [reply content/en/user-manual/optimization/mini-stats.md]: A portion of our developers won't be familiar with `requestAnimationFrame`. So how about: `the total time in milliseconds for the browser to process each frame`.

Comment (slimbuck) [reply content/en/user-manual/optimization/mini-stats.md]: done

**Pull request merged**